### PR TITLE
Use Updated colors library

### PR DESCRIPTION
**Pull request by mdedys**

### CODE REVIEW

File: bar-chart.html [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <link rel='import' href='../polymer/polymer.html'>
-<link rel='import' href='../vui-colors/colors.html'>
-<link rel='import' href='../vui-typography/typography.html'>
+<link rel='import' href='../d2l-colors/d2l-colors.html'>
```
Comment (kieranderson) [inline bar-chart.html]: There is a style tag around line 7, that references vui-colors. Also, are all the variables still using ```vui``` at the start?

Comment (mdedys) [inline bar-chart.html]: Good catch. I was just about to test in the FRA. Changed to d2l-colors

Comment (mdedys) [inline bar-chart.html]: Should we just remove the dependency on d2l-colors and just use the value? We only use 1 color from it anyways? What do you think?

**Comment by kieranderson** [comment]
:+1:

**Pull request merged**